docs/supported_formats/jobin_yvon.rst

```diff
@@ -0,0 +1,56 @@
+.. _jobin_yvon-format:
+
+Horiba Jobin Yvon LabSpec
+-------------------------
+
+Reader for spectroscopy data saved using Horiba Jobin Yvon's LabSpec software.
+Currently, RosettaSciIO can only read the ``.xml`` format from Jobin Yvon.
+However, this format supports spectral maps and contains all relevant metadata.
+Therefore, it is a good alternative to the binary ``.l5s`` or ``.l6s`` formats
+in order to transfer data to python or other analysis software.
+
+If `LumiSpy <https://lumispy.org>`_ is installed, ``LumiSpectrum`` will be
+used as the ``signal_type``.
+
+When working with `HyperSpy <https://hyperspy.org>`_, a file can be read using
+the following code:
+
+.. code-block:: python
+
+    >>> import hyperspy.api as hs
+    >>> sig = hs.load("file.xml", reader="JobinYvon")
+
+Specifying the reader is necessary as the :ref:`EMPAD format <empad-format>`
+also uses the ``.xml`` file-extension.
+
+The reader supports all signal axis units exported by LabSpec, i.e. wavelengths,
+wavenumbers (absolute), Raman shift (relative wavenumbers),
+as well as energy. 
+
+.. Note::
+
+  The wavelength-to-energy conversion is not documented for LabSpec, i.e. it is
+  not clear whether the permittivity of air is taken into account. When working
```

Comment:
From what I see, the term "permittivity of air" is largely used in the codebase, e.g.:

https://github.com/LumiSpy/lumispy/blob/4669de6846c30ab40b65ee0308b1b7812b96f5b3/lumispy/utils/axes.py#L34

This is fine but as far as I can see, correction is applied using the real-valued refractive index. Just something to be aware of...

Reply:
Indeed, we should be consistent and use `refractive index of air`. For the other occurences I have opened #147.

Reply:
Nice. Thank you for this,

Reply:
I changed that for jobin_yvon.rst.